packages/ethereum-contracts/test/foundry/utils/MacroForwarder.t.sol

```diff
@@ -87,6 +97,20 @@ contract MultiFlowDeleteMacro is IUserDefinedMacro {
             });
         }
     }
+
+    function postCheck(ISuperfluid host, bytes memory params, address msgSender) external view { }
+}
+
+contract MacroWithRevertingPostCheck is IUserDefinedMacro {
+    function buildBatchOperations(ISuperfluid, bytes memory, address /*msgSender*/) external override pure
+        returns (ISuperfluid.Operation[] memory /*operation*/)
+    {
+        return new ISuperfluid.Operation[](0);
+    }
+
+    function postCheck(ISuperfluid host, bytes memory params, address msgSender) external pure {
+        revert("I'm a bad macro");
```

Comment:
I would make it based on "params" passed, and have a positive and negative test cases to show how postCheck should be used.

Reply:
> I would make it based on "params" passed, and have a positive and negative test cases to show how postCheck should be used.

done, please check

Reply:
okay.

Actually, on the second thought, a more "realistic" use case of  `postCheck` should be checking the "effect" of the `bool retVal = _forwardBatchCall(operations);`

Because checking params could have been done during `buildBatchOperations` too.

We can leave that example to docs or inline comments, instead. It's up to you how much you want to spend more time in this PR.